Dockerfile

```diff
@@ -1,12 +1,23 @@
-ARG node_version
-FROM node:$node_version
-
-WORKDIR /usr/app
-
-COPY package.json .
-COPY package-lock.json .
-
+#base Build
+ARG node_version=10
```

Comment:
Actually, that's a [default value](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact) in case the actual is not provided.   It is more like a fail-safe mechanism.   Docker build uses the ARG if provided and if not it uses the default.